JUnit version 4.12
.E....
Time: 1.862
There was 1 failure:
1) test0(org.joda.time.chrono.ISOChronology_ESTest)
org.joda.time.IllegalFieldValueException: Value 3559 for hourOfDay must be in the range [0,23]
	at org.joda.time.field.FieldUtils.verifyValueBounds(FieldUtils.java:236)
	at org.joda.time.chrono.BasicChronology.getDateTimeMillis(BasicChronology.java:172)
	at org.joda.time.chrono.GregorianChronology.getDateTimeMillis(GregorianChronology.java:45)
	at org.joda.time.chrono.AssembledChronology.getDateTimeMillis(AssembledChronology.java:133)
	at org.joda.time.chrono.ZonedChronology.getDateTimeMillis(ZonedChronology.java:119)
	at org.joda.time.chrono.AssembledChronology.getDateTimeMillis(AssembledChronology.java:133)
	at org.joda.time.chrono.ISOChronology_ESTest.test0(ISOChronology_ESTest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)

FAILURES!!!
Tests run: 5,  Failures: 1

